src/main/java/frc/robot/subsystems/swerve/states/DriveSpeed.java

```diff
@@ -0,0 +1,24 @@
+package frc.robot.subsystems.swerve.states;
+
+public enum DriveSpeed {
+
+	NORMAL(1, 1),
+	SLOW(0.5, 0.5);
```

Comment:
only a suggestion - I think slow should be slower (use case is mostly safety) , up to your decision